src/main/validation/DTOConverter.ts

```diff
@@ -0,0 +1,77 @@
+import { User } from '../entity/User';
+import { Promotion } from '../entity/Promotion';
+import { ScheduleDTO } from './ScheduleValidation';
+import { PromotionDTO } from './PromotionValidation';
+import { Schedule } from '../entity/Schedule';
+import { Discount } from '../entity/Discount';
+import { DiscountDTO } from './DiscountValidation';
+import { UserDTO } from './UserValidation';
+
+/**
+ * Converts DTOs to entity's
+ * */
+export class DTOConverter {
+  /**
+   * Convert PromotionDTO to a Promotion
+   */
+  static promotionDTOtoPromotion(
```

Comment:
Just curious but is there a particular reason why you chose static fns?

Reply:
Good question! This class doesn't really have any "state" to maintain so I didn't really see a point in instantiating it and injecting it into `PromotionController` and `UserController`. We could've just removed the whole class, and exported each as a function, but yah I didn't give much thought to it.

Reply:
Oooh cool I see!! That makes sense